src/temp-data-component.js

```diff
@@ -0,0 +1,134 @@
+let allNodes = [{
+    nodeId: 0,
+    name: 'N0',
+    x: 300,
+    y: 150,
+    content: 'blank0',
+}, {
+    nodeId: 1,
+    name: 'N1',
+    x: 140,
+    y: 300,
+    content: 'blank1',
+}, {
+    nodeId: 2,
+    name: 'N2',
+    x: 300,
+    y: 300,
+    content: 'blank2',
+}, {
+    nodeId: 3,
+    name: 'N3',
+    x: 300,
+    y: 180,
+    content: 'blank3',
+}, {
+    nodeId: 4,
+    name: 'N4',
+    x: 300,
+    y: 180,
+    content: 'blank4',
+}, {
+    nodeId: 5,
+    name: 'N5',
+    x: 300,
+    y: 180,
+    content: 'blank5',
+}, {
+    nodeId: 6,
+    name: 'N6',
+    x: 300,
+    y: 180,
+    content: 'blank6',
+}]
+
+let allLinks = [{

```

Comment:
`const` > `let`

Reply:
this list will eventually be modifiable when we add creation/deletion of nodes

Reply:
'this list' should not be modifiable. it defeats the purpose of managing state the way it is currently managed.

I think what you're wanting to do is load this list from the filesystem at load time, place it in the state object, performs actions to update the state (nodes, links data) as required. Variables, especially top level variables in a module, have no reason (that I can see) to be mutable in such as use case.